tests/testthat/test-type-vec-size-common-error.txt

```diff
@@ -1,10 +1,10 @@
 
 vec_size_common(1:2, 1, 1:4):
 
-No common size for `..1`, size 2, and `..3`, size 4. 
+Can't recycle `..1` (size 2) and `..3` (size 4). 
```

Comment:
This seems like a strange error to me. I think recycling and finding a common size are different enough that this shouldn't say anything about "recycle". I view recycling as taking `x` and extending it to have size `size`. It isn't an operation between `x` and `y` directly.

Maybe:

```
Can't compute a common size among inputs.
* `..1` has size 2.
* `..3` has size 4. 
```

If `x_arg = ""` I'm not quite sure what you'd do here.

`vec_recycle_common()` would use the same error because an error would only occur there when computing a common size, but `vec_recycle()` would be different:

```
Can't recycle input `x` to size 4.
* It must have size 1 or 4.
* It has size 3.
```

Reply:
The goal is to make it consistent with vec-cast-common and vec-cast. vec-recycle would eventually match the same form.

I originally had "combine" instead of "recycle" but @hadley preferred to mention recycling.